lib/cli/commands/restore.ts

```diff
@@ -28,7 +44,47 @@ const formatOutput = (response: RestoreNodeResponse.AsObject) => {
   console.log(walletRestoredMessage);
 };
 
-export const handler = (argv: Arguments<any>) => {
+export const handler = async (argv: Arguments<any>) => {
+  const request = new RestoreNodeRequest();
+
+  if (argv.backup_directory && argv.backup_directory !== '') {
```

Comment:
I don't think you need the empty check because if so the first part would be false.